lnpeer/peer.go

```diff
@@ -25,7 +51,9 @@ type Peer interface {
 
 	// AddNewChannel adds a new channel to the peer. The channel should fail
 	// to be added if the cancel channel is closed.
-	AddNewChannel(channel *channeldb.OpenChannel, cancel <-chan struct{}) error
+	AddNewChannel(channel *channeldb.OpenChannel,
```

Comment:
Don't think this interface method needs to be expanded to care more than the `*channeldb.OpenChannel`. Since we can access db in `brontide`, we can instead query the policy from db.